verilog/formatting/formatter_test.cc

```diff
@@ -1767,6 +1790,29 @@ static const std::initializer_list<FormatterTestCase> kFormatterTestCases = {
         "// verilog_syntax: parse-as-module-body\n"
         "`define FOO\n",
     },
+    {// test alternate parsing mode in macro expansion
+     "class foo;\n"
+     "`MY_MACRO(\n"
+     " // verilog_syntax: parse-as-statements\n"
+     " // EOL comment\n"
+     " int count;\n"
+     " if(cfg.enable) begin\n"
+     "   count = 1;\n"
```

Comment:
small suggestion: un-indent `count = 1` here to show that it does get properly indented in the expected output.

Reply:
Done.